packages/collector/test/tracing/protocols/apollo_subgraph/test.js

```diff
@@ -18,16 +17,12 @@ const globalAgent = require('../../../globalAgent');
 const agentControls = globalAgent.instance;
 
 const mochaSuiteFn = supportedVersion(process.versions.node) ? describe : describe.skip;
-
-mochaSuiteFn('tracing/apollo-federation', function () {
+mochaSuiteFn('tracing gateway with apollo-subgraph', function () {
   this.timeout(config.getTestTimeout() * 2);
```

Comment:
Could you pls increase the timeout? The test is failing on my machine. I had to increase it to

> this.timeout(config.getTestTimeout() * 5);